### PR TITLE
opensuse migration then rollback and zdup fix

### PR DESCRIPTION
**Pull request by mitiao**
see: https://progress.opensuse.org/issues/17836
Followed by #2587 

verified runs:
http://147.2.207.208/tests/463
http://147.2.207.208/tests/470

### CODE REVIEW

File: tests/installation/setup_zdup.pm [hunk 1]
```diff
@@ -31,6 +31,14 @@ sub run() {
...
 
         select_console('root-console');
+        # Create a snapshot with specified description to do snapper rollback
+        # This action is concerned about following points:
+        # 1. Source image could be original installation or updated
+        # 2. Source image may apply patches before migration
+        # 3. Hard to assert similar snapshots in grub2
+        # 4. Menu of each snapshot is long with openSUSE leap, use short and unique description
+        # 5. Avoid rollback to snapshot without graphical target
+        assert_script_run "snapper create --type pre --cleanup-algorithm=number --print-number --userdata important=yes --description 'b_zdup migration'";
```
Comment (mitiao) [inline tests/installation/setup_zdup.pm]: please see https://progress.opensuse.org/issues/18318#note-4

File: products/opensuse/main.pm [hunk 2]
```diff
@@ -632,8 +632,10 @@ sub load_x11tests() {
             loadtest "x11/chrome";
         }
     }
-
-    loadtest "x11/shutdown";
+    # Need to skip shutdown to keep beckend alive if run rollback tests after migration
```
Comment (okurz) [inline products/opensuse/main.pm]: s/beckend/backend/; s/run/running/

File: tests/installation/setup_zdup.pm (hunk 1 above)
Comment (okurz) [inline tests/installation/setup_zdup.pm]: yes, fine. It's only in "zdup" case, I did not realize this before.

File: products/opensuse/main.pm (hunk 2 above)
Comment (mitiao) [inline products/opensuse/main.pm]: replaced.

**Pull request merged**